docs/pages/accounts/sso.mdx

```diff
@@ -22,6 +22,8 @@ SSO is available for [Enterprise Plan](https://expo.dev/pricing) customers. To g
 
 ### Expo CLI
 
+> Available from SDK 50. For previous versions, the Expo CLI will use your SSO account after logging in via the EAS CLI.
```

Comment:
```suggestion
> Only available with SDK 50+. For previous versions, the Expo CLI will use your SSO account after logging in via the EAS CLI.
```

Reply:
@keith-kurak, let's update this to something "SDK 50 and above" (in general, we should avoid using symbols as per [this writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#symbols-as-words)